frontend/public/components/namespace.jsx

```diff
@@ -26,6 +26,7 @@ import { OverviewNamespaceDashboard } from './overview/namespace-overview';
 const getModel = useProjects => useProjects ? ProjectModel : NamespaceModel;
 const getDisplayName = obj => _.get(obj, ['metadata', 'annotations', 'openshift.io/display-name']);
 const getRequester = obj => _.get(obj, ['metadata', 'annotations', 'openshift.io/requester']);
+const CREATE_NEW_PROJECT = 'create-new-project';
```

Comment:
Not unique enough as I can create a namespace with this name. It must be a unique key that will not clash with a namespace value.

Reply:
updated the key